Dfe.Data.SearchPrototype/Web/Tests/DfE.Data.SearchPrototype.Pages/HttpHelpers.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace DfE.Data.SearchPrototype.Pages;
+
+public static class HttpHelpers
+{
+    public static async Task<Response> GetHttpResponseAsync(this HttpClient client, string path)

```

Comment:
Neccessity to share this across
`PresentationLayerTests`
`IntegrationTests` 

both using WebApplicationFactory to creae a HttpResponse that is parsed. Added as an extension for now.